kolibri/plugins/user_profile/assets/src/views/ChangeFacility/MergeUsernameExists.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('documentTitle') }}</h1>
```

Comment:
@jredrejo It would be better to add this(`$tr('documentTitle')`) as a computed constant under the `setup()` hook since [ Line:48](https://github.com/learningequality/kolibri/pull/9687/files#diff-b6d35063aff073cfabb167aec1a23a7fc14dc458fd3e26b5204784eb8eb1d9f4R48) has the same implementation. This would improve maintainability of the translated title

Reply:
I don't think that would improve maintainability as the function is already implicit in the tr variable. Both strings don't have to be the same, they're in this design but not in all the screens. 
Also adding a computed const won't work to be used in both places.

Reply:
Thanks @jredrejo for the context. My initial impression was that its the same string required in both places, so there was no need to do the translation twice.